source/kraken/apply_disruption.cpp

```diff
@@ -182,6 +182,21 @@ static std::string concatenate_impact_uris(const nt::MetaVehicleJourney& mvj) {
     return impacts_uris.str();
 }
 
+
+static nt::Route* get_or_create_route(const nt::disruption::Impact& impact, nt::PT_Data& pt_data) {
```

Comment:
```suggestion
static nt::Route* get_or_create_route(nt::PT_Data& pt_data) {
```

Reply:
I like the compiler warning, it underlines the fact that we have to use impact and create meaningful objects (route, etc.).
From what I understood of our discussion, it's OK to keep it that way.